tests/api/routes/auth.spec.js

```diff
@@ -11,7 +11,7 @@ describe('Auth Endpoint', () => {
     })
     it('will allow a user to login', async () => {
       const uniqueEmail = `testuser${Date.now()}@gmail.com`
-      const teacher = await createUser(uniqueEmail, 'password', 'TEACHER')
+      // const teacher = await createUser(uniqueEmail, 'password', 'TEACHER')
```

Comment:
Should these comments be reverted to their original state, or are they unnecessary?
if they are unnecessary better to delete them